OpenRA.Test/OpenRA.Game/CPosTest.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System.Runtime.Remoting.Metadata;
-using NUnit.Framework;
+﻿using NUnit.Framework;
```

Comment:
This file appears to be missing our standard license header.  Would you mind adding that while you are here?

Reply:
Sure, ill do another run in a few hours adding it to all files where it is missing.